### DISCUSSION TITLE
Expanded messaging roles with JMAP?

### DISCUSSION THREAD

**Discussion by forrie** [jmapio/jmap#358] [Ideas]
I am new to JMAP, so this may likely be a question answered elsewhere.  However, I wonder if JMAP could also be used to potentially encompass lower-end messaging (ie: instant messaging, etc.) as a unifying protocol?

**Comment by neilj** [comment]
Yes, you could use it for instant messaging or IRC/Slack style chat rooms. Someone would just need to champion that and bring an initial draft to the IETF if they wanted to standardise it. I don’t think anyone is working on that at the moment, but I think it could be interesting in the future!
